apps/storybook/.storybook/preview.tsx

```diff
@@ -1,5 +1,5 @@
-import './globals.css'
 import '../../website/src/globals.css'
+import './globals.css'
```

Comment:
When these were imported the other way around there was a case of `flex-col` overriding `sm:flex-row`. This is because `flex-col` was defined in './globals.css' but `sm:flex-row` wasn't, so it had precedence. Importing them the other way around should ensure that the styling is the same as when running the project directly

Reply:
Tailwind prunes out classes that aren't used (in a pretty naive way, just by matching exact string). I'm not exactly sure how the compilation works in storybook, but my guess is that `import './globals.css'` only includes a few base classes and classes that are used in the actual `apps/storybook` project, so it doesn't include `sm:flex-row`.

That may not be it but either way this does empirically fix the problem 😌 .